### ISSUE TITLE
Local Flow Plugins Not Loading

### ISSUE THREAD

**Issue by nichols89ben** [HaveAGitGat/Tdarr#1093]
Cant get local flow plugins to load in one of my docker compose servers. Both servers are on  2.26.01 and have identical plug in folders, its loading in one but not the other. Thoughts?

![Server 1 Tree](https://github.com/user-attachments/assets/e0b26ef8-6434-407b-ae01-af8c1a21fd89)
![Server 2 Tree](https://github.com/user-attachments/assets/04201d24-35f2-496a-9b29-46db646ad0cc)
![Server 1](https://github.com/user-attachments/assets/ceff41bf-a523-45dd-b7eb-ddddf98a4b7f)
![Server 2](https://github.com/user-attachments/assets/b948cbe4-98b6-4462-9a49-6f921ec7f268)

**Comment by HaveAGitGat** [comment]
If they're identical but it's working in one and not the other then that would suggest maybe file permissions? Have you checked this? Try copying a community flow plugin into local to see if that works.

**Comment by nichols89ben** [comment]
> If they're identical but it's working in one and not the other then that would suggest maybe file permissions? Have you checked this? Try copying a community flow plugin into local to see if that works.

Permissions was the first thing I tried and reapplied them for the whole folder and that didn't fix it. I just rechecked now and noticed all of the local plugins missing in the server with the issues, so I re added them and they're populating now. Not sure what the issue was as they where all there and I restarted/re-pulled the server a few times to verify. Must be a tdarr ghost.
